tests/utils_tests/test_lazyobject.py

```diff
@@ -288,6 +288,26 @@ def test_deepcopy_class_no_evaluation(self):
         self.assertIs(obj._wrapped, empty)
         self.assertIs(obj2._wrapped, empty)
 
+    def test_getattribute(self):
+        attrs = [
```

Comment:
Is an accurate docstring something like `"""Proxy methods don't exist on wrapped objects unless they're set."""`?